docs/releasenotes/9.1.0.rst

```diff
@@ -18,11 +18,34 @@ Rather than returning a ``SystemError``, passing the incorrect types of coordina
 a path will now raise a more specific ``ValueError``, with the message "incorrect
 coordinate type".
 
+Replace requirements.txt with extras
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Rather than installing all dependencies for docs and tests via ``requirements.txt``,
+``extras_require`` is used instead. This installs only those needed and at the same
+time as installing Pillow.
+
+For example:
+
+.. code-block:: bash
+
+    # Install with dependencies for tests:
+    python3 -m pip install .[tests]
+
+    # Or for building docs:
+    python3 -m pip install .[docs]
+
+    # Or for all:
+    python3 -m pip install .[docs,tests]
+
```

Comment:
```suggestion

On macOS, the last argument must be wrapped in quotes, e.g. `python3 -m pip install ".[tests]"`

```

My suggestion to document [this.](https://github.com/python-pillow/Pillow/pull/6072#issue-1144747636) But feel free to reword and improve it.
> \# Or my Mac terminal needs quotes
> pip install ".[docs]"

Reply:
Yes, let's include something. I left it out as I'm not sure if it's a "must be" or "may need to be" depending on the terminal, are they needed on your Mac too?

(Also needs double backticks for RST.)

Reply:
They are needed on my Mac.
I've updated the suggestion to say "may", and with double backticks.

Reply:
Thanks!